### ISSUE TITLE
My HealtheVet - Document sign-in user flows post 9/19/24 updates to MHV NP

### ISSUE THREAD

**Issue by sterkenburgsara** [department-of-veterans-affairs/va.gov-team#93410]
## Overview

On 9/19/24 some updates were pushed to the national portal regarding the change management and pivot away from the MHV sign-in credential. Some changes may have an impact on whether or not our access alerts or back links from VA.gov still make sense and route to the correct places on the national portal. 

### Team

- Requester: Sara Sterkenburg [Pending prioritization by POs]
- Team Name: Cartography

### Description
> What details are necessary for understanding the specific work or request tracked by this issue?
See [parent ticket here ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/93408)

### Links

[MURAL](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1707497060514/1b011002bf25fbfe1f1c832c0566296ed86c0bb9?sender=uf94a77a19aaf687331c09367)

### Work needed

> What outcome do you anticipate the fulfillment of this request to deliver?
- User flows are documented and any areas with an issue are flagged for follow-on work to fix those issues. 

### Timeline
> When does IA review need to be completed by? ASAP - should be considered urgent

## Acceptance Criteria
- [x] Evaluate sign-in process on National Portal and document user flows
- [x] Evaluate sign-in process on VA.gov and document user flows
  * LOA1 ID.me and Login.gov flows 
  * LOA1 MHV basic flow
  * LOA3 "not registered with My HealtheVet" flow
  * LOA3 "no access" flow
  * LOA3 - has full access flow

**Comment by NaomiPMC** [comment]
@sterkenburgsara  is there a difference between this ticket and #93408 or can we simply work from one ticket, if so, which ticket should that be?

**Comment by sterkenburgsara** [comment]
@NaomiPMC apologies for not seeing this earlier! The broader ticket also impacts @laurwill - we both need to understand what has changed in the IA of the MHV National Portal in order to determine if the flows we intended are still possible with changed URLs/UX on that side. If that is not possible, we may need to amend content in alerts/on-page messaging within health tool pages.

This narrower ticket was intended just for me specifically (IA) to document those user flows between VA.gov and the MHV National Portal to the extent I can (the staging environment has not been a good proxy lately).

**Comment by sterkenburgsara** [comment]
Update: we have bandaided up a few problems that were identified with URL routing from identity-verification flow alerts and "go back" links to the national portal (ID.me, Login.gov) a couple of weeks ago. These changes followed some adjustments in the national portal to eliminate the creation of My HealtheVet basic accounts as part of the sign-in transition work to sunset this credential entirely by January 2025. The outstanding items that still need to be addressed are: 

* LOA3 "not registered with My HealtheVet" flow - the link in this alert does send a user to the national portal, where a user can scroll down to fill out the registration form. However, there is a problematic page title (H1) on the page that isn't a clear match to the CTA that was in the link text (instead of saying "Register for My HealtheVet" it says "Continue to My HealtheVet") and the page requires scrolling past a banner alert on the national portal that doesn't make sense for this scenario. I'm not sure how to influence these types of UX changes on the national portal, so I started[ this Slack thread last week](https://dsva.slack.com/archives/C0581MN69TJ/p1728673039655039). 
  * OCTO stakeholders are referring me to Carnetta and her team for follow-up, which I will do once PI planning is wrapped and they are available to respond. 
* Several of these alerts outlined in the ticket body above need to be updated to match the versions [just finalized last week](https://dsva.slack.com/archives/C068QM7CCKG/p1728493055673339) - we will wait for Design System Team to publish these alerts (in code, not just in Figma) and pull in those instances as soon as we are able.
